src/ConfirmationTooltip/index.tsx

```diff
@@ -0,0 +1,78 @@
+import "../../node_modules/tippy.js/animations/shift-away.css";
+import React from "react";
+import Tippy from "@tippy.js/react";
+import { TippyStyles } from "../shared/TippyStyles";
+import { useSpaceKitProvider } from "../SpaceKitProvider";
+
+type TippyProps = React.ComponentProps<typeof Tippy>;
+
+interface Props
+  extends Pick<TippyProps, "children" | "content">,
+    Partial<Pick<TippyProps, "placement">> {
+  /**
+   * Force the tooltip to be visible. Only use this for testing purposes; don't
+   * use this to programatically control the component
+   */
+  forceVisibleForTestingOnly?: boolean;
```

Comment:
Is this a different take on naming scheme vs. something like `__testVisible`?

Reply:
Yes. I was hoping that `ForTestingOnly` would be more obvious than `_testVisible` to those unaware of our AGM practices.